Cargo.toml

```diff
@@ -50,6 +50,7 @@ mio = "0.6"
 libc = "0.2"
 lazy_static = "1.4.0"
 url = "2.1.1"
+percent-encoding = "2.1.0"
```

Comment:
This is part of the `url` crate source package.  Should we use the same version number for it and `url`?

Reply:
Sure -- that'll mean downgrading `url` to "2.1.0", as there's no 2.1.1 for percent-encoding. But it'll have more or less the same meaning as  "2.1.1", since these all are implicitly using semver "^" comparisons.